packages/sanity/src/desk/structureBuilder/Intent.ts

```diff
@@ -19,6 +19,12 @@ export interface BaseIntentParams {
   id?: string
   /* Intent template */
   template?: string
+  /**
+   * Experimental field path
+   * @beta
+   * @experimental
```

Comment:
Same comment as I made in a separate place, given that it's @beta & @experimental do we want it showing right now or should we add @hidden ?